@xen-orchestra/lite/src/components/pool/system/PoolStorageConfiguration.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <UiCard>
+    <UiTitle>
+      {{ $t('storage-configuration') }}
+    </UiTitle>
+    <VtsLoadingHero v-if="!isReady" type="card" :title="$t('storage-configuration')" />
+    <template v-else>
+      <VtsQuickInfoRow :label="$t('default-storage-repository')">
+        <template #value>
+          <UiLink v-if="defaultSr" v-tooltip="$t('coming-soon')" :icon="faDatabase" size="medium">
+            {{ defaultSr.name_label }}
+          </UiLink>
```

Comment:
For all the links in this component, you can make the same changes as we discussed for XO 6:
```suggestion
          <template v-if="defaultSr">
            <VtsIcon :icon="faDatabase" accent="current" />
            {{ defaultSr.name_label }}
          </template>
```